polytracker/src/compiler-rt/include/CMakeLists.txt

```diff
@@ -0,0 +1,104 @@
+if (COMPILER_RT_BUILD_SANITIZERS)
+  set(SANITIZER_HEADERS
+    # sanitizer/allocator_interface.h
+    # sanitizer/asan_interface.h
+    # sanitizer/common_interface_defs.h
+    # sanitizer/coverage_interface.h
+    sanitizer/dfsan_interface.h
+    # sanitizer/hwasan_interface.h
+    # sanitizer/linux_syscall_hooks.h
+    # sanitizer/lsan_interface.h
+    # sanitizer/msan_interface.h
+    # sanitizer/netbsd_syscall_hooks.h
+    # sanitizer/scudo_interface.h
+    # sanitizer/tsan_interface.h
+    # sanitizer/tsan_interface_atomic.h
+    # sanitizer/ubsan_interface.h
+    )
+  set(FUZZER_HEADERS
+    fuzzer/FuzzedDataProvider.h
+    )
+endif(COMPILER_RT_BUILD_SANITIZERS)
+
+if (COMPILER_RT_BUILD_MEMPROF)
+  set(MEMPROF_HEADERS
+    sanitizer/memprof_interface.h
+    )
+endif(COMPILER_RT_BUILD_MEMPROF)
+
+if (COMPILER_RT_BUILD_XRAY)
+  set(XRAY_HEADERS
+    xray/xray_interface.h
+    xray/xray_log_interface.h
+    xray/xray_records.h
+    )
+endif(COMPILER_RT_BUILD_XRAY)
+
+if (COMPILER_RT_BUILD_PROFILE)
+  set(PROFILE_HEADERS
+    profile/InstrProfData.inc
+    )
+endif(COMPILER_RT_BUILD_PROFILE)
+
+set(COMPILER_RT_HEADERS
+  ${SANITIZER_HEADERS}
+  # ${FUZZER_HEADERS}
+  # ${MEMPROF_HEADERS}
+  # ${XRAY_HEADERS}
```

Comment:
I'm not sure. Since we built a modified version of the whole `compiler-rt` previously, we did have XRAY included in it. The `compiler-rt` excerpt from this PR doesn't have it included. Theoretically it should be possible to link against an `apt` installed version of `compiler-rt` but I haven't tried that.